mix.exs

```diff
@@ -37,15 +37,16 @@ defmodule Membrane.RTP.MPEGAudio.MixProject do
   defp docs do
     [
       main: "readme",
-      extras: ["README.md"],
+      extras: ["README.md", "LICENSE"],
```

Comment:
The license will have weird, capitalized name (IIRC) unless you do
```suggestion
      extras: ["README.md", LICENSE: [title: "License"]],
```

Reply:
It's not added to https://github.com/membraneframework/membrane_template_plugin